audiovario.c

```diff
@@ -76,58 +77,62 @@ float change_volume(float delta){
   return volume;
 }
 
-float pulse_syn(float phase, float rise, float fall, float duty ){
+inline float pulse_syn(float phase, float rise, float fall, float risei, float falli, float duty ){
```

Comment:
What these new variables are for?

Reply:
The previous code divided by rise and fall.  The new code multiplies by the reciprocal.  I was assuming that was faster.  Perhaps not?